docs-src/concepts/what-is-versioning.md

```diff
@@ -0,0 +1,170 @@
+
+Rather than creating a new Workflow Type for each version of your Workflow, you can create a new Task Queue for each version of your Workflow.
+
+This approach is similar to Workflow Type versioning, but instead of creating a new Workflow Type for each version of your Workflow, you create a new Task Queue for each version of your Workflow.
+
+This helps maintain clean libraries without branching logic.
+
+### Advantages of Task Queue-based Versioning
+
+Task Queue-based Versioning is valuable in several respects:
+
+- **Conceptually easy**: Task Queues are similar to Workflow Type versioning, but with Task Queues, so you don't need to manually update all starters to use the latest Workflow Type.
+- **Easy maintenance**: In comparison to Patching, Task Queue-based Versioning requires less maintenance because you don't need to track which Workflows are running on which version of the Workflow Type.
+
+### Disadvantages of Task Queue-based Versioning
+
+Task Queue Versioning has several disadvantages:
+
+- **Operationally complex**: To run multiple versions of a Task Queue, you need to keep old Workers alive and change which Task Queue Clients point to. This can result in a number of sets of old Workers, which operationally can be complex to manage.
+- **Bug fixes**: This approach can't be used to fix a bug in a currently running or open Workflow. Because the Workflow is already running, you are unable to change the Task Queue it's running on, and thus you can't make changes or fixes to the Workflow.
+
+## Patching
+
+Patching allows for the update of Workflow Definitions without affecting currently running Workflows.
+
+Patching, is a set of steps or a technique used to manage changes or upgrades to your Workflow’s logic. Because Workflows can run for weeks, months, or years, you may want to change your Workflow code over time with changes that aren't backwards compatible.
+
+Take the example of needing to modify some business logic that oversees the shipment of an item. Terminating all Workflows midway through delivery would not be a desirable approach. Instead, you would use patching to update your Workflow code.
+
+:::note
+
+In the [Go](https://pkg.go.dev/go.temporal.io/sdk/workflow#GetVersion) and [Java](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/workflow/Workflow.html#getVersion(java.lang.String,int,int)) SDKs, Patching is referred to as `GetVersion`.
+
+:::
+
+### What is Patching
+
+Patching allows you to update your Workflow code without terminating any running Workflows and retaining compatibility with your Workflow Event History, as a way to manage changes or upgrades to your Workflow’s logic.
+
+### Advantages of Patching
+
+Patching is valuable in several respects:
+
+- **No Downtime**: Patching allows you to update your Workflow code without terminating any running Workflows.
```

Comment:
With this wording, this is true of _all_ the solutions. I'd just strike this line.